April CQA work for MTA 8.1 Release Notes DITA fixes (#356)
migtools/mta-documentation: docs/topics/release-notes-topics/ref_new-features-and-enhancements-8-0.adoc

Reviewer: ```suggestion
NOTE: You must restart your server after changing the GenAI settings for the change to take effect.
```

For consistency

= New features and enhancements

[role="_abstract"]

> Review the following major new features and enhancements in {ProductFirstRef} ({ProductShortName}) 8.0.0.

Reviewer 2: ```suggestion
Review new features and enhancements in {ProductFirstRef} ({ProductShortName}) 8.0.0.
```

+
link:https://issues.redhat.com/browse/MTA-4846[MTA-4846]

{ProductShortName} supports asset generation for application migration in the web console::
You can use the discovery manifest to generate assets in the web console to deploy applications in Red Hat OpenShift or Kubernetes. The asset generation workflow includes generators, target profiles, and the archetypes. Generators parse the application manifest configurations to create the deployment assets for a specific kind of target platform. In the target profiles, you can run one or more generators in sequence for a given archetype.
+
link:https://issues.redhat.com/browse/MTA-4847[MTA-4847]

{ProductShortName} CLI supports running a live discovery in a remote CF instance::

> With this update, you can perform a live discovery to determine applications that you deployed in a certain Cloud Foundry (CF) cluster. For example, you can determine how many applications are in the cluster. You can also use the live discovery if you do not have access to manifest YAML files.

Reviewer 2: This way it's enough. 
```suggestion
With this update, you can perform a live discovery to determine applications deployed in a certain Cloud Foundry (CF) cluster. For example, you can determine how many applications are in the cluster. You can also use the live discovery if you do not have access to manifest YAML files.
```

{ProductShortName} CLI supports running a live discovery in a remote CF instance::

> With this update, you can perform a live discovery if you want to determine applications that you deployed in a certain Cloud Foundry (CF) cluster. For example, you can determine how many applications are in the cluster. You can also use the live discovery if you do not have access to manifest YAML files.

Reviewer: "if you want to determine applications that" part can be simplified.

```suggestion
With this update, you can perform a live discovery to determine applications deployed in a certain Cloud Foundry (CF) cluster. For example, you can determine how many applications are in the cluster. You can also use the live discovery if you do not have access to manifest YAML files.
```

Author: I understand but I think "applications deployed in a certain..." is passive voice?

--

Analysis insights are available in the {ProductShortName} UI::
Analysis insights contain information about the technologies used in the application and their usage in the code. Before this update, you could only view insights in the analysis output and a static report after running an application analysis. With this enhancement, you can access analysis insights also from the {ProductFullName} user interface (UI).
+
(link:https://issues.redhat.com/browse/MTA-5420[MTA-5420])

A new Visual Studio Code IDE plugin for application analysis::
With this enhancement, you can use a new Visual Studio Code Integrated Development Environment (IDE) plugin that has the following features:

* Standard application analysis and issue detection
* Generative AI (GenAI) feature:

> ** When enabled, you can use GenAI to find solutions for the analysis incidents. Note that {mta-dl-plugin} enabled GenAI by default.

Reviewer: This is probably another example of active voice which works better as passive. Consider changing it back. Fix if it occurs elsewhere too.

```suggestion
** When enabled, you can use GenAI to find solutions for the analysis incidents. Note that GenAI is enabled by default.
```

+
--
NOTE: You must restart your server after changing the GenAI settings for the change to take effect.

link:https://issues.redhat.com/browse/MTA-5360[MTA-5360]
--

{ProductShortName} UI supports setting default credentials::

> Before this update, you could only assign credentials to an application manually after you added or imported this application to the {ProductFullName} user interface (UI). However, you might need to import a large set of applications,with each application requiring credentials for language and technology discovery tasks. With this update, instead of manually defining credentials, you can define default credentials for any application that does not yet have credentials.

Reviewer 2: ```suggestion
Before this update, you could only assign credentials to an application manually after you added or imported this application to the {ProductFullName} user interface (UI). However, you might need to import a large set of applications, with each application requiring credentials for language and technology discovery tasks. With this update, instead of manually defining credentials, you can define default credentials for any application that does not have credentials.
```

--

{ProductShortName} UI supports setting default credentials::

> Before this update, you could only assign credentials to an application manually once you added or imported this application to the {ProductFullName} user interface (UI). However, you might need to import a large set of applications, each of these applications requiring credentials for language and technology discovery tasks. With this update, instead of manually defining credentials, you can define default credentials for any application that does not yet have credentials.

Reviewer: "each of these applications requiring" could be simplified

```suggestion
Before this update, you could only assign credentials to an application manually once you added or imported this application to the {ProductFullName} user interface (UI). However, you might need to import a large set of applications, each requiring credentials for language and technology discovery tasks. With this update, instead of manually defining credentials, you can define default credentials for any application that does not yet have credentials.
```

Author: I think "with each application requiring..." here gives more clarity and is consistent with your earlier suggestions?

+

> IMPORTANT: You can only set Maven or source control credentials as default credentials. You can only define one default set of credentials per credential type.

Reviewer: Place 'only' immediately before the word it modifies.

```suggestion
IMPORTANT: You can only set Maven or source control credentials as default credentials. You can define only one default set of credentials per credential type.
```
